h2o-py/tests/testdir_misc/pyunit_mojo_predict.py

```diff
@@ -155,8 +155,8 @@ def mojo_predict_csv_test(target_dir):
     prediction_result = h2o.mojo_predict_csv(input_csv_path=input_csv, mojo_zip_path=mojo_zip_path,
                                              output_csv_path=output_csv)
 
-    mojo_prediction_0 = float(prediction_result[0]['0'])
-    mojo_prediction_1 = float(prediction_result[0]['1'])
+    mojo_prediction_0 = float(prediction_result[0]['p0'])
+    mojo_prediction_1 = float(prediction_result[0]['p1'])
     print("Binomial prediction: p0: " + str(mojo_prediction_0))
```

Comment:
@michalk:  Will this break customer code?

Reply:
yes, this is a breaking change - people will have to adapt to it - should be noted in the migration document